### PR TITLE
task_6

### PR DESCRIPTION
**Pull request by ArgonTrime**
1 task, work with, Promis, async await

### CODE REVIEW

File: task_6/1.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// запрос на покемон, получить urls всех покемонов, получить массив объектов покемонов по urls
+
+const getPokemons = async (url) => {
+    const pokemonsArray = [];
+    const pokemonsUrl = [];
```
Comment (Boobaley) [inline task_6/1.js]: what the purpose of this?

File: task_6/1.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    try {
+        const {results} = await fetch(url).then((response) => response.json());
+        results.forEach(({url}) => pokemonsUrl.push(url));
```
Comment (Boobaley) [inline task_6/1.js]: this is redundant. you can use map method. No need to collect urls in separate variable

File: task_6/1.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+        await Promise.allSettled(pokemonsUrl.map(url => fetch(url).then(responce => responce.json())))
+        .then((responces) => {
+            responces.forEach(({value}) => pokemonsArray.push(value));
```
Comment (Boobaley) [inline task_6/1.js]: const result = await Promise.all(...)? why did you create separate variable to save the result above?

**Review by Boobaley — CHANGES_REQUESTED** [review]

**Comment by ArgonTrime** [comment]
add 2, 3 task and fix first task with Promise

**Pull request merged**